data/base-hunting.yaml

```diff
@@ -407,6 +407,31 @@ hunting_zones:
   - 1446
   - 1447
   - 1448
+  silverfish:                                                             
+  # https://elanthipedia.play.net/Silverfish                              10-32
+  # Do Not Use 6079.  It is a terrible quadruplicate.
```

Comment:
What is a terrible quadrublicate?

Reply:
theres four rooms that are dups in mapdb and cannot be mapped better than what I did.  should never hunt there.  it's also noted on a new map image i have pending

Reply:
Just say why? Also this is a data file, it feels wierd to leave notes like this in my opinion

Reply:
Updated